doc/irb/indexes.rdoc

```diff
@@ -0,0 +1,193 @@
+  initial value: +nil+.
+- <tt>:LC_MESSAGES</tt>: {Locale}[rdoc-ref:IRB@Locale];
+  initial value: IRB::Locale object.
+- <tt>:LOAD_MODULES</tt>: deprecated.
+- <tt>:MAIN_CONTEXT</tt>: The {context}[rdoc-ref:IRB@Session+Context] for the main \IRB session;
+  initial value: IRB::Context object.
+- <tt>:MEASURE</tt>: Whether to
+  {measure performance}[rdoc-ref:IRB@Performance+Measurement];
+  initial value: +false+.
+- <tt>:MEASURE_CALLBACKS</tt>: Callback methods for
+  {performance measurement}[rdoc-ref:IRB@Performance+Measurement];
+  initial value: <tt>[]</tt>.
+- <tt>:MEASURE_PROC</tt>: Procs for
+  {performance measurement}[rdoc-ref:IRB@Performance+Measurement];
+  initial value:
+
+    {
+      :TIME=>#<Proc:0x0000556e271c6598 /var/lib/gems/3.0.0/gems/irb-1.8.3/lib/irb/init.rb:106>,
+      :STACKPROF=>#<Proc:0x0000556e271c6548 /var/lib/gems/3.0.0/gems/irb-1.8.3/lib/irb/init.rb:116>
+    }
+
+- <tt>:PROMPT</tt>: \Hash of {defined prompts}[rdoc-ref:IRB@Prompt+and+Return+Formats];
+  initial value:
+
+    {
+      :NULL=>{:PROMPT_I=>nil, :PROMPT_S=>nil, :PROMPT_C=>nil, :RETURN=>"%s\n"},
+      :DEFAULT=>{:PROMPT_I=>"%N(%m):%03n> ", :PROMPT_S=>"%N(%m):%03n%l ", :PROMPT_C=>"%N(%m):%03n* ", :RETURN=>"=> %s\n"},
+      :CLASSIC=>{:PROMPT_I=>"%N(%m):%03n:%i> ", :PROMPT_S=>"%N(%m):%03n:%i%l ", :PROMPT_C=>"%N(%m):%03n:%i* ", :RETURN=>"%s\n"},
+      :SIMPLE=>{:PROMPT_I=>">> ", :PROMPT_S=>"%l> ", :PROMPT_C=>"?> ", :RETURN=>"=> %s\n"},
+      :INF_RUBY=>{:PROMPT_I=>"%N(%m):%03n> ", :PROMPT_S=>nil, :PROMPT_C=>nil, :RETURN=>"%s\n", :AUTO_INDENT=>true},
+      :XMP=>{:PROMPT_I=>nil, :PROMPT_S=>nil, :PROMPT_C=>nil, :RETURN=>"    ==>%s\n"}
+    }
+
+- <tt>:PROMPT_MODE</tt>: Name of {current prompt}[rdoc-ref:IRB@Prompt+and+Return+Formats];
+  initial value: +:DEFAULT+.
+- <tt>:RC</tt>: Whether a {configuration file}[rdoc-ref:IRB@Configuration+File]
+  was found and interpreted;
+  initial value: +true+ if a configuration file was found, +false+ otherwise.
+- <tt>:RC_NAME_GENERATOR</tt>: \Proc to generate paths of potential
+  {configuration files}[rdoc-ref:IRB@Configuration+File];
+  initial value: <tt>=> #<Proc:0x000055f9bebfed80 /var/lib/gems/3.0.0/gems/irb-1.8.3/lib/irb/init.rb:401></tt>.
+- <tt>:SAVE_HISTORY</tt>: Number of commands to save in
+  {input command history}[rdoc-ref:IRB@Input+Command+History];
+  initial value: +1000+.
+- <tt>:SINGLE_IRB</tt>: Whether command-line option <tt>--single-irb</tt> was given;
+  initial value: +true+ if the option was given, +false+ otherwise.
+  See {Single-IRB Mode}[rdoc-ref:IRB@Single-IRB+Mode].
+- <tt>:USE_AUTOCOMPLETE</tt>: Whether to use
+  {automatic completion}[rdoc-ref:IRB@Automatic+Completion];
+  initial value: +true+.
+- <tt>:USE_COLORIZE</tt>: Whether to use
+  {color highlighting}[rdoc-ref:IRB@Color+Highlighting];
+  initial value: +true+.
+- <tt>:USE_LOADER</tt>: Whether to use the
+  {IRB loader}[rdoc-ref:IRB@IRB+Loader] for +require+ and +load+;
+  initial value: +false+.
```

Comment:
I recently added `USE_PAGER` as well. It controls if pager is enabled for commands and return value output and defaults to `true`.